tools/ci_build/build.py

```diff
@@ -1391,7 +1391,7 @@ def build_python_wheel(
         use_acl, nightly_build=False, featurizers_build=False):
     for config in configs:
         cwd = get_config_build_dir(build_dir, config)
-        if is_windows():
+        if is_windows() and not os.path.exists(os.path.join(cwd, 'build.ninja')):
```

Comment:
args.cmake_generator == 'Ninja' would be a more direct test.

Reply:
args is not passed into this function, do you think it would be better to add a parameter use_ninja to build_python_wheel?

Reply:
yes. thanks for making the change. it's better to test directly as there could potentially be output from an old build and the build.ninja file could exist from that. 

---
In reply to: [436105679](https://github.com/microsoft/onnxruntime/pull/4135#discussion_r436105679) [](ancestors = 436105679)